api_tests/nodes/views/test_node_contributors_list.py

```diff
@@ -952,7 +962,7 @@ def test_add_unregistered_contributor_with_fullname(
             project_public.reload()
             assert res.status_code == 201
             assert res.json['data']['attributes']['unregistered_contributor'] == 'John Doe'
-            assert res.json['data']['attributes']['email'] is None
+            assert res.json['data']['attributes'].get('email') is None
```

Comment:
Minor: no need for the code to be defensive in tests. Better to be...assertive 😅 ... about what is expected. In other words, if `email` isn't expected to be in `attributes`, test that directly.

```
assert 'email' not in res.json['data']['attributes']
```